utilities/rali/rali_dataloader_mt/README.md

```diff
@@ -0,0 +1,25 @@
+# rali_dataloader_mt application
+This application demonstrates a basic usage of RALI's C API to use sharded data_loader  in a multithreaded application.
+<p align="center"><img width="90%" src="../../docs/images/image_augmentation.png" /></p>
+
+## Build Instructions
+
+### Pre-requisites
+*   Ubuntu Linux, [version `16.04` or later] (https://ubuntu.com/download)
```

Comment:
Codacy found an issue: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=5745979)